app/build.gradle

```diff
@@ -28,6 +28,9 @@ android {
     }
     kotlinOptions {
         jvmTarget = '1.8'
+        freeCompilerArgs += [
+                "-Xjvm-default=all",
```

Comment:
Why add this attribute ?

Reply:
https://stackoverflow.com/questions/70992947/how-do-i-resolve-error-message-inheritance-from-an-interface-with-jvmdefault 

I faced the error as this question and solved it by adding this attribute